MANIFEST

```diff
@@ -2995,6 +2999,23 @@
 ./utils/test/Makefile.am
 ./utils/test/swmr_check_compat_vfd.c
 
+# parallel tools (dh5walk) and tests
+./utils/tools/CMakeLists.txt
+./utils/tools/Makefile.am
+./utils/tools/dh5walk/CMakeLists.txt
+./utils/tools/dh5walk/Makefile
```

Comment:
Makefile is not a source file:  should be removed from MANIFEST

Reply:
Fixed in my latest commit..  Thank you!